testing/py-html5lib/APKBUILD

```diff
@@ -1,41 +1,49 @@
 # Contributor: Fabian Affolter <fabian@affolter-engineering.ch>
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=py-html5lib
 _pkgname=html5lib
-pkgver=0.999
+pkgver=1.0.1
 pkgrel=0
 pkgdesc="A Python HTML parser"
 url="https://github.com/html5lib/html5lib-python"
 arch="noarch"
 license="MIT"
-depends="python2"
-depends_dev=""
-makedepends="python2-dev py-setuptools"
-install=""
-subpackages=""
+options="!check"
```

Comment:
there is test suite https://github.com/html5lib/html5lib-python#tests

Reply:
Adding missing dependencies to run tests.